test/integration/targets/ansible-doc/collections/ansible_collections/testns/testcol/plugins/filter/ultimatequestion.yml

```diff
@@ -1,19 +1,19 @@
 DOCUMENTATION:
   name: ultimatequestion
   author: Terry Prachet
-  version_added: 'histerical'
-  short_description: Ask any question but it will only respond with the answer to the ulitmate one
+  version_added: 'historical'
```

Comment:
this one was actually on purpose ...

Reply:
Yeah, just noticed that, too. I'll update the PR.

Reply:
And fixed.